test_suite/README.md

```diff
@@ -25,172 +26,157 @@ These tests are used to validate ports of the API.
+Some of the tests has been annotated with a pytest marker:
 
-Run the command `python -m pytest -m <marker name>` to run the test anotated with `marker name`.
+- `ble41`: Test of Bluetooth 4.1 features.
+- `ble42`: Test of Bluetooth 4.2 features.
+- `ble50`: Test of Bluetooth 5.0 features.
+- `smoketest`: Small set of smoke tests.
 
-## Select targets type
+To run the test annotated with `marker name`, run the command `pytest -m <marker name>`.
 
-To select specific target types that should run the test, use the command line 
-argument `--platforms=`. Multiple platforms can be specified, they must be separated 
-with a `,`. 
+## Select targets
 
-For example, this command runs tests on `NRF52840_DK`: 
+To select specific targets for the test, use the command line
+argument `--platforms=`. To specify multiple platforms, separate them with a comma `,`.<!--this isn't a type - it's a specific board, isn't it?-->
+
+For example, to run tests on `NRF52840_DK`:
 
 ```sh
-python -m pytest --platforms=NRF52840_DK
+pytest --platforms=NRF52840_DK
 ```
 
-
 ## Flash targets
 
-It is possible to flash targets with a binary before the test start. Use the 
-argument `--flash=`. A list of platform/binary pair can be defined using the syntax
-`platform:binary`. Multiple pairs can be specified, they must be separated 
-with a `,`. 
+To flash targets with a binary before the test starts, use `--flash=`.<!--do I have to? is that part of the steps you described above for test running? or is it if I have something else I need on the board, and I can skip this test?-->
 
-For example, this following command run tests on `NRF52840_DK` and flash the binary
-`ble-cliapp.hex` present in the working directory: 
+You can define a list of platform/binary pairs using the syntax
+`platform:binary`. To specify multiple pairs, separate them with a comma `,`.
+
+For example, to run tests on `NRF52840_DK` and flash the binary
+`ble-cliapp.hex` (already in the working directory):<!--doesn't it first flash and then run the tests?-->
 
 ```sh
 pytest --flash=NRF52840_DK:ble-cliapp.hex
 ```
 
 ********************************************************************************
 
-# Extending the test suite 
-
-Mbed OS BLE test suite is built using pytest. We recomand reading the exhaustive 
-pytest [documentation](https://docs.pytest.org/en/stable/) when specific problems 
-not covered by this document need to be solved. 
+# Extending the test suite
 
+the Mbed OS BLE test suite is built using pytest. We recommend reading the [pytest documentation](https://docs.pytest.org/en/stable/) to solve problems
+not covered by this document.
 
-## Test conventions 
+## Test conventions
 
-Test files must start with the name `test` to be discovered by pytest. 
-Similarly, test functions name must start with the prefix `test`. 
+To be discovered by pytest:
 
+- Test files must start with the name `test`.<!--name here, prefix in the next line - are they really different things?-->
```

Comment:
Yes they are, both conditions should be met for pytest to discover tests. It first looks at files that starts with the name `test` then inside these files it identify tests function if it begins with `test`.

Reply:
I meant, "prefix" and "name starts with". Do we really need two different words?

Reply:
I understand now, we don't need two different words.

Reply:
```suggestion
- Test files must start with the prefix `test`.<!--name here, prefix in the next line - are they really different things?-->
```